app/controllers/answers_controller.rb

```diff
@@ -0,0 +1,66 @@
+class AnswersController < ApplicationController
+
+  def create
+    @question = Question.find_by(id: params[:question_id])
+    @answer = Answer.new(answer_params)
```

Comment:
Think about doing @answer = @question.answers.new / build to make the association clear to the reader immediately.